kitsune/users/tests/test_models.py

```diff
@@ -1,11 +1,10 @@
 import logging
 
-from nose.tools import eq_
-
 from kitsune.sumo.tests import TestCase
 from kitsune.users.forms import SettingsForm
 from kitsune.users.models import Setting
 from kitsune.users.tests import UserFactory
+from nose.tools import eq_
```

Comment:
this should be reverted to the previous stage. First python libs followed by django libs and external libs and finally our import paths.